extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmConfigPersistenceUnit.java

```diff
@@ -512,6 +513,93 @@ default boolean isAnyPropertySet() {
 
     }
 
+    enum HibernateTimeZoneStorageType {
+        /**
+         * Stores the timezone by using the {@code with time zone}
+         * SQL column type.
+         * <p>
+         * Error if {@link Dialect#getTimeZoneSupport()} is not
+         * {@link org.hibernate.dialect.TimeZoneSupport#NATIVE}.
+         *
```

Comment:
You need to us Asciidoc if you use `@asciidoclet`. 

```suggestion
         * Stores the timezone by using the `with time zone`
         * SQL column type.
         * 
         * Error if {@link Dialect#getTimeZoneSupport()} is not
         * {@link org.hibernate.dialect.TimeZoneSupport#NATIVE}.
         *
```

I'm not sure about `@link`, better check out how it's rendered.